graphdatascience/pipeline/nr_training_pipeline.py

```diff
@@ -9,24 +9,61 @@
 
 class NRTrainingPipeline(TrainingPipeline[NRModel]):
     def addLinearRegression(self, **config: Any) -> "Series[Any]":
+        """
+        Add a linear regression model to the pipeline.
```

Comment:
```suggestion
        Add a linear regression model candidate to the pipeline.
```
In the server docs we call these candidates for clarity